modules/efk-logging-fluentd-collector.adoc

```diff
@@ -8,6 +8,7 @@
 {product-title} cluster logging uses Fluentd by default. 
 Log collectors are deployed as a DaemonSet to each node in the cluster. 
 
+////
```

Comment:
Sorry. This file is hidden and the user shouldn't be able to access it. ButI added your wording and commented-out the procedure just in case someone comes across it.